PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/AzureRunProfileState.java

```diff
@@ -46,26 +52,41 @@ public ExecutionResult execute(Executor executor, @NotNull ProgramRunner program
         ConsoleView consoleView = TextConsoleBuilderFactory.getInstance().createBuilder(this.project).getConsole();
         processHandler.startNotify();
         consoleView.attachToProcess(processHandler);
+
+        Hooks.onErrorDropped(ex -> {
```

Comment:
As we are adding a global config, shall we move this part to `AzurePlugin` or other startup listener to keep consistent behavior?